pkg/logql/quantile_over_time_sketch_test.go

```diff
@@ -188,3 +189,38 @@ func (ev *sliceStepEvaluator) Next() (ok bool, ts int64, r StepResult) {
 	ok = ev.cur < len(ev.slice)
 	return
 }
+
+func BenchmarkQuantileBatchRangeVectorIteratorAt(b *testing.B) {
```

Comment:
I was planning to change the way we aggregate as well. I think this level is a little bit better. It would also test `tdigest` if we chose that one instead.

Reply:
The name is still unclear as to why the mapping change results in an improvement here, and the PR description is a bit vague too.

Is this description accurate?
> The term mapping refers to how DDSketch decides whether a given floating point value is a part of one or another bucket within the approximated histogram buckets.

If so, wouldn't a benchmark that inserts things into the sketch be a more accurate test of performance for the mapping?

Reply:
The description is about but a little too much into the detial IMHO. We should care about the performance of `At`.